src/test/java/org/jenkinsci/plugins/docker/workflow/client/DockerClientTest.java

```diff
@@ -62,7 +62,7 @@ public void test_run() throws IOException, InterruptedException {
         EnvVars launchEnv = newLaunchEnv();
         String containerId =
                 dockerClient.run(launchEnv, "learn/tutorial", null, null, Collections.<String, String>emptyMap(), Collections.<String>emptyList(), new EnvVars(),
-                        dockerClient.whoAmI(), "true");
+                        dockerClient.whoAmI(), "cat");
```

Comment:
This test was bogus as `true` was returning immediately, so the container was not even running.